python/lsst/ts/scheduler/coldstart/config.py

```diff
@@ -0,0 +1,14 @@
+# Absolute folder path that contains all your databases
```

Comment:
I don't think it is a good idea to have this kind of information on a package. Specially, note how the paths are basically pointing to local files on your own machine, which will definitively change if you are on a different computer or system. This should probably become command line arguments to some script that uses it.